samples/js/node/pronunciationAssessment.js

```diff
@@ -0,0 +1,297 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+// pull in the required packages.
+var sdk = require("microsoft-cognitiveservices-speech-sdk");
+const filePushStream = require("./filePushStream");
+const _ = require('lodash') ;
+const forEach = require('lodash.foreach');
+const Segment = require('segment');
+var difflib = require('difflib');
+
+
+(function() {
+"use strict";
+
```

Comment:
add the following comment

`pronunciation assessment with audio streaming and continue mode`

Reply:
added

Reply:
Please also show the streaming progress in your code.

Reply:
update to display recognition process